requirements/prod.txt

```diff
@@ -9,4 +9,4 @@ PyYaml>=5.1,<6.0
 sceptre-cmd-resolver>=1.1.3,<2
 sceptre-file-resolver>=1.0.4,<2
 six>=1.11.0,<2.0.0
-troposphere>=2.0.0,<2.1.0
+troposphere>=2.7,<3
```

Comment:
@zaro0508 recently in slack, you mentioned `2.0.2` works.  Should this be `>=2.0.2,<3` or are there versions greater than `2.0.2` and less than `2.7` that cause issues?

Reply:
sorry, i think the slack conversation was a bit confusing.  To clarify, I get the `AttributeError` on a troposphere template deployment when run with ver 2.0.2.  I reran the deployment with troposphere ver 2.7.1 and it works without error.  I did not test with any other versions.

Reply:
got it